modules/order/config/schema/commerce_order.schema.yml

```diff
@@ -17,6 +17,12 @@ commerce_order.commerce_order_type.*:
     refresh_frequency:
         type: integer
         label: 'Order refresh frequency'
+    sendReceipt:
+        type: boolean
+        label: 'Send an email to the customer when the order is placed'
+    receiptBcc:
```

Comment:
We should use this opportunity to make receiptBcc a setting for the actual email used. We lose nothing, it's additional flexibility. helps us with the getter too, shouldAddReceiptBcc -> getReceiptBcc, much better.